static/js/tippyjs.js

```diff
@@ -8,6 +8,16 @@ import * as reactions from "./reactions";
 import * as rows from "./rows";
 import * as timerender from "./timerender";
 
+function get_tooltip_content(reference) {
+    if ("tooltipId" in reference.dataset) {
```

Comment:
Can we add a comment documenting this function, ideally containing the string `data-tooltip-template-id` here for greppability?

Reply:
Note that this function is not exported, it’s only used in a place that already has such a comment, and really we should just learn how the web platform works instead of trying to paper over it…but sure, I guess.

Reply:
It doesn't matter if we learn it; we have over 1000 people who've contributed to Zulip at this point and constantly new ones arriving, and there's approximately infinite things to know about the web platform, so it's worth putting bits of effort into keeping the codebase readable when using patterns that are not widespread in the codebase.